nixos/modules/services/web-servers/hydron.nix

```diff
@@ -0,0 +1,118 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.hydron;
+in
+{
+  options.services.hydron = {
+    enable = mkEnableOption "hydron";
+
+    baseDir = mkOption {
+      type = types.path;
+      default = "/var/lib/hydron";
+      description = "Location where hydron runs.";
+    };
+
+    listenAddress = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      description = "Listen on a specific IP address and port.";
```

Comment:
Can you add an explanation of which format this is expected to be in? `<ip>:<port>` I assume?